python/pyspark/graphx/partitionstrategy.py

```diff
@@ -0,0 +1,9 @@
+__author__ = 'kdatta1'
```

Comment:
We don't use author tags, so I'd remove this.

Also, this file and a few others need the Spark license header.